RELEASING.md

```diff
@@ -1,8 +1,12 @@
-1. Make certain your branch is in sync with head
+1. Create a new virtual environment following the instructions in `CONTRIBUTING.md`. 
+   Make sure to also install all dependencies for the documentation including `altair_saver`
+   and uninstall `vl-convert-python`.
```

Comment:
```suggestion
   and uninstall `vl-convert-python` (not needed for normal contributions).
```
Every now and then I use this page to find out what I need to do to build the docs locally. Maybe leave a small note here, that altair_saver is really only needed for releasing and not _standard_ contributions? Or a reference to the new section for contributing on docs?